packages/extract-react-types/src/index.js

```diff
@@ -1,8 +1,7 @@
 // @flow
-/*::
-export type * from './kinds'
-import * as K from './kinds'
-*/
+import * as K from './kinds';
+
+export type * from './kinds';
```

Comment:
Now have a mixture of `module.exports` and `export`. Probably just have to bite the bullet and babel this to a cjs dist?